ethportal-peertest/src/scenarios/bridge.rs

```diff
@@ -31,3 +31,26 @@ pub async fn test_bridge(peertest: &Peertest, target: &HttpClient) {
         "The received content {received_content_value:?}, must match the expected {content_value:?}",
     );
 }
+
+pub async fn test_bridge_yaml(peertest: &Peertest, target: &HttpClient) {
```

Comment:
I agree with @carver that this functionality should be tested, but I somewhat disagree with how. There's no need to actually run the entire bridge scenario again. All the test needs to do is validate that [`TestAssets`](https://github.com/ethereum/trin/blob/master/portal-bridge/src/bridge.rs#L83-L84) can be created from both a yaml and json file. After moving the decoding logic as @ogenev suggested, I would add a test to the utils file that simply validates that either a `yaml` or `json` file can be parsed into the `TestAssets` type.

Reply:
resolved